microservices/biobuddy/biobuddy/services.py

```diff
@@ -0,0 +1,21 @@
+from biobuddy.api_models import SendMessageToBioBuddyResponse, SendMessageToBioBuddyRequest
+
+from openai import OpenAI
+
+__all__ = ['send_message']
+
+client = OpenAI()
+
+
+def send_message(request: SendMessageToBioBuddyRequest) -> SendMessageToBioBuddyResponse:
+    messages = request.previous_messages
+    messages.append({"role": "system",
+             "content": "You are a biobuddy. You are a research assistant that helps creating new drugs, researching biology, pulling information from different sources and running experiments."
+                        "If a user asks you to pull the targets only do so for proteins and use rcsb pdb. If a user asks for drugs/ligands them use chembl. If a user asks for latest research about the illness then use pubmed."})
+    messages.append( {"role": "user", "content": request.message_content})
+    completion = client.chat.completions.create(
```

Comment:
Not critical, but I would rather move this stuff to config files

Reply:
Let's do later when we have more function calls, now we need the speed for the release early next week